test/fixtures/bad-url.result.html

```diff
@@ -1 +1 @@
-<img src="data:undefined;base64,">
```

Comment:
I'm not sure if this is bad or not. I couldn't quite figure out why it's getting a HTML page out of the sudden.